index.html

```diff
@@ -8771,8 +8780,8 @@ <h5>Presentational Roles Conflict Resolution</h5>
 		<div class="role" id="term">
 			<rdef>term</rdef>
 			<div class="role-description">
-				<p>A word or phrase with a corresponding definition. See related <rref>definition</rref>.</p>
-				<p>The <code>term</code> role is used to explicitly identify a word or phrase for which a <rref>definition</rref> has been provided by the author or is expected to be provided by the user.</p>
+				<p>A word or phrase with an optional corresponding definition. See related <rref>definition</rref>.</p>
+				<p>The <code>term</code> role is used to explicitly identify a word or phrase for which a <rref>definition</rref> has been provided by the author or is expected to be provided by the user. Where a matching <rref>definition</rref> exists, authors MUST set <pref>aria-details</pref> on the <code>term</code>, pointing to the <rref>definition</rref>. Otherwise, <pref>aria-details</pref> MUST point to a form or control that take user input that provides the definition.</p>
```

Comment:
regarding:
>MUST point to a form or control that take user input that provides the definition

* that "takes" user input
* could this not also point to a link that takes users to the definition?

i like this idea though.

Reply:
It could take the user to a link with the definition, but I'm not sure we really have to say that.